app/src/main/java/com/github/dfa/diaspora_android/ui/theming/ColorPalette.java

```diff
@@ -0,0 +1,288 @@
+package com.github.dfa.diaspora_android.ui.theming;
+
+import android.content.Context;
+import android.graphics.Color;
+import android.support.v4.content.ContextCompat;
+import android.support.v4.graphics.ColorUtils;
+
+import com.github.dfa.diaspora_android.R;
+
+/**
+ * Created by dnld on 24/02/16.
+ */
+public class ColorPalette {
```

Comment:
I looked around for some time, but I couldn't come up with something that makes sense. What exactly do you have in mind?